### PR TITLE
[AppKit] Fix intro tests on monterey.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Review by rolfbjarne — APPROVED** [review]

**Review by chamons — COMMENTED** [review]
Did you really mean https://github.com/xamarin/xamarin-macios/issues/1318 from Dec 8, 2016?

**Comment by mandel-macaque** [comment]
@chamons  typo, should be https://github.com/xamarin/xamarin-macios/issues/13185 got fixed

File: src/appkit.cs [hunk 1]
```diff
@@ -809,13 +809,14 @@ interface NSApplication : NSAccessibilityElementProtocol, NSUserInterfaceValidat
 		[Notification, Field ("NSApplicationDidChangeScreenParametersNotification")]
 		NSString DidChangeScreenParametersNotification { get; }
 
-		[Mac (12,0)]
-		[Field ("NSApplicationProtectedDataWillBecomeUnavailableNotification")]
-		NSString ProtectedDataWillBecomeUnavailableNotification { get; }
+		// https://github.com/xamarin/xamarin-macios/issues/1318
```
Comment (mandel-macaque) [inline src/appkit.cs:812]:
```suggestion
		// https://github.com/xamarin/xamarin-macios/issues/13185
```

**Review by chamons — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Pull request merged**